packages/oidc-dpop-client-browser/rollup.config.js

```diff
@@ -1,9 +1,5 @@
 import pkg from "./package.json";
 import typescript from "rollup-plugin-typescript2";
-import resolve from "@rollup/plugin-node-resolve";
-import builtins from 'rollup-plugin-node-builtins';
-import commonjs from "@rollup/plugin-commonjs";
-import polyfills from 'rollup-plugin-node-polyfills';
```

Comment:
The linter should be catching these things, once you're using it (https://github.com/inrupt/javascript-style-configs/tree/master/eslint-config-lib)